client/css/style.css

```diff
@@ -280,6 +281,10 @@ kbd {
 #sidebar .chan.special::before,
 #chat .special .title::before { content: "\f03a"; /* http://fontawesome.io/icon/list/ */ }
 
+.chan::before {
+	padding-top: 3px;
```

Comment:
Note, this is here because I noticed the icons for the server and channels were actually slightly misaligned.

|Before|After|
|-|-|
|![](https://files.yamanickill.com/Ym3n6edo.png)|![](https://files.yamanickill.com/ZX-Tfij5.png)|

Reply:
Hmm, to me the server icon looks misaligned in the before image, but the channel icon looks misaligned in the after image. Good eye though. I didn't notice the alignment at all before.

Reply:
Yeah, you might be right. I'll change it to just do the lobby and see how that looks.

Reply:
Alright, just pushing up a change to only do this for the lobby icon.

![](https://files.yamanickill.com/bEbM-MS0.png)